demos/basic/main.cpp

```diff
@@ -33,12 +33,12 @@
 
 static std::string const testFile(TULPAR_DEMO_WORKING_DIR"data/ding_02.ogg");
 
-constexpr double pi()
+double pi()
 {
     return std::atan(1) * 4;
 }
 
-constexpr double tau()
+double tau()
```

Comment:
Would it be useful to have both versions available?

Reply:
GCC and clang supply `constexpr std::atan` whereas CL doesn't. Using `constexpr variable` instead.